packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -603,8 +849,233 @@ export const Tabs = (props: TabsProps) => {
+  'button',
+  TabLegacyProps & { value?: string }
+>;
+
+/**
+ * Tabs organize and allow navigation between groups of content that are related and at the same level of hierarchy.
+ * @example
+ * <Tabs>
+ *   <Tabs.TabList>
+ *     <Tabs.Tab value='tab1' label='Label 1' />
+ *     <Tabs.Tab value='tab2' label='Label 2' />
+ *     <Tabs.Tab value='tab3' label='Label 3' />
+ *   </Tabs.TabList>
+ *   <Tabs.ActionsWrapper>
+ *     <Tabs.Actions>
+ *       <Button>Sample Button</Button>
+ *     </Tabs.Actions>
+ *   </Tabs.ActionsWrapper>
+ *   <Tabs.Panel value='tab1'>Content 1</Tabs.Panel>
+ *   <Tabs.Panel value='tab2'>Content 2</Tabs.Panel>
+ *   <Tabs.Panel value='tab3'>Content 3</Tabs.Panel>
+ * </Tabs>
+ *
+ * @example
+ * <Tabs orientation='vertical'/>
+ *
+ * @example
+ * <Tabs.Tab value='sample'>
+ *   <Tabs.TabIcon>
+ *     <SvgPlaceholder />
+ *   </Tabs.TabIcon>
+ *   <Tabs.TabLabel>Sample Label</Tabs.TabLabel>
+ *   <Tabs.TabDescription>Sample Description</Tabs.TabDescription>
+ * </Tabs.Tab>
+ *
+ */
+
+export const Tabs = Object.assign(TabsComponent, {
+  /**
+   * A wrapper component for Tabs
+   */
+  Wrapper: TabsWrapper,
+  /**
+   * Tablist subcomponent which contains all of the tab subcomponents.
+   * @example
+   * <Tabs.TabList>
+   *   <Tabs.Tab value='tab1' label='Label 1' />
+   *   <Tabs.Tab value='tab2' label='Label 2' />
+   *   <Tabs.Tab value='tab3' label='Label 3' />
+   * </Tabs.TabList>
+   *
+   * @example
+   * <Tabs.TabList color='green'>
+   *   <Tabs.Tab value='tab1' label='Green Tab' />
+   * </Tabs.TabList>
+   *
+   * @example
+   * <Tabs.TabList focusActivationMode='manual'>
+   *   <Tabs.Tab value='tab1' label='Manual Focus Tab' />
+   * </Tabs.TabList>
+   */
+  TabList: TabList,
+  /**
+   * Tab subcomponent which is used for each of the tabs.
+   * @example
+   * <Tabs.Tab value='tab1' label='Label 1' />
+   *
+   * @example
+   * <Tabs.Tab value='sample'>
+   *   <Tabs.TabIcon>
+   *     <SvgPlaceholder />
+   *   </Tabs.TabIcon>
+   *   <Tabs.TabLabel>Sample Label</Tabs.TabLabel>
+   *   <Tabs.TabDescription>Sample Description</Tabs.TabDescription>
+   * </Tabs.Tab>
+   *
+   */
+  Tab: TabHeader,
+  /**
+   * Tab icon subcomponent which places an icon on the left side of the tab.
+   */
+  TabIcon: TabIcon,
+  /**
+   * Tab label subcomponent which holds the tab's label.
+   */
+  TabLabel: TabLabel,
+  /**
+   * Tab description subcomponent which places a description under the tab label.
+   */
+  TabDescription: TabDescription,
+  /**
+   * Tab actions subcomponent which contains action buttons that are placed at the end of the tabs.
+   */
+  Actions: TabsActions,
+  /**
+   * Tab panel subcomponent which contains the tab's content.
+   * @example
+   * <Tabs.Panel value='tab1'>
+   *   Sample Panel
+   * </Tabs.Panel>
+   */
+  Panel: TabsPanel,
+});
+
+export const TabsContext = React.createContext<
+  | {
+      /**
+       * Type of the tabs.
+       */
+      type?: string;
```

Comment:
Does the reason why this is a `string` and not actual values have something to do with the `type` values can vary based on the orientation?

Reply:
Updated the type